internal/cli/compile/compile.go

```diff
@@ -355,13 +363,19 @@ type compileResult struct {
 	Success       bool                 `json:"success"`
 	ProfileOut    string               `json:"profile_out,omitempty"`
 	Error         string               `json:"error,omitempty"`
+
+	showOnlyProperties bool
 }
 
 func (r *compileResult) Data() interface{} {
 	return r
 }
 
 func (r *compileResult) String() string {
+	if r.showOnlyProperties {
```

Comment:
Should we introduce a different type of result? Like `ShowPropertiesResult` with the same interface
The information contained in the json and plaintext format have some, meaningless, differences already and they might require to get back at it in future